ui_advanced_config.py

```diff
@@ -203,26 +220,44 @@ def build_ui(self):
             master=self.componentsColumn, command=self.delete_components, text="Delete"
         )
         self.componentsIDLabel = uiLabel(master=self.componentsColumn, text="ID:")
```

Comment:
I'm not sure if it matters, but the help buttons of non-active component entries are clickable and provide some sort of content. we should double check that the info given for comonent entries works properly between different component types (i.e. when max_cmds_per_tick help box is selected, it should not give info on reload_ticks)

Reply:
the correct help buttons are clickable now, but some appeared grey-ed out even when they should be/are clickable
<img width="351" alt="Screen Shot 2024-04-04 at 1 07 52 AM" src="https://github.com/Leading-AI-Development-Solutions/MAIA/assets/100381732/df81c26d-3abe-4077-84c9-ac1857e8f925">

Reply:
Still occurring for me as well

Reply:
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/123680493/851f07d1-1908-4b4b-826b-e3b08ded1b99)

for what parts graham?

Reply:
It seems to be an issue where the button looks disabled but it isn't. The functionality is correct, but the color isn't updated sometimes. Seems like it might be tkinter limitation?

Reply:
Oh what the heck, it works for me now. Just a minute ago it looked like Sophie's screenshot for me, but now it looks like yours. Bad luck ig lol

Reply:
I'm still looking into it because it shouldn't randomly look disabled when they aren't. As far as I'm aware I'm enabling/disabling buttons in the correct way, so idk what the best fix would be for this problem.

Reply:
Pushed a potential fix. I seem to no longer get the issue, but since it seems to happen randomly it is worth other people testing it.

Reply:
i can't get it to happen again, and i tried multiple times. it's possible that if it was appearing at random previously that this or a similar issue might pop up in the future, but if we can't replicate i'm fine with calling this fixed.